packages/stark-ui/src/modules/action-bar/components/action-bar.component.html

```diff
@@ -4,7 +4,7 @@
 		<div *ngFor="let action of actionBarConfig.actions; trackBy: trackAction" [id]="actionBarId + '-' + action.id"
 		(click)="onClick(action, $event)" class="stark-action-bar--action">
 			<ng-container *ngIf="action.isVisible !== false">
-				<button [color]="action.buttonColor ? action.buttonColor:'primary'" [ngClass]="action.className" [matTooltip]="action.labelSwitchFunction ? action.labelActivated : action.label" [disabled]="!action.isEnabled" mat-icon-button>
+				<button [color]="action.buttonColor ? action.buttonColor:'primary'" [ngClass]="action.className" [matTooltip]="action.labelSwitchFunction ? action.labelActivated : action.label | translate" [disabled]="!action.isEnabled" mat-icon-button>
```

Comment:
Just a question: are we sure this syntax works together with the `translate` pipe? Are both possible values correctly translated?
`"action.labelSwitchFunction ? action.labelActivated : action.label | translate"`